config/application.rb

```diff
@@ -10,7 +10,7 @@
 require "action_mailer/railtie"
 require "action_view/railtie"
 require "action_cable/engine"
-# require "sprockets/railtie"
+ require "sprockets/railtie"
```

Comment:
Having this commented was causing some errors with graphql, so I put it back in.

Reply:
Nice Work!